ncclient/operations/third_party/iosxr/rpc.py

```diff
@@ -0,0 +1,13 @@
+from ncclient.xml_ import *
+
+from ncclient.operations.rpc import RPC
+from ncclient.operations.rpc import RPCReply
+from ncclient.operations.rpc import RPCError
+from ncclient import NCClientError
+
+class ExecuteRpc(RPC):
+    def request(self, rpc, filter_xml=None):
+        if isinstance(rpc, str):
```

Comment:
What happens if I pass something apart from a string? Should the string be checked for inclusion of bad characters? E.g. a string looking like "<rpc-name/>" will be accepted. What happens if an lxml.etree object is passed in? Etc.

Need better error checking at minimum.